srml/support/src/storage/generator.rs

```diff
@@ -636,9 +636,9 @@ mod tests {
 			GETMAPU32MYDEF get(map_u32_getter_mydef): map u32 => String = "map".into();
 			pub PUBGETMAPU32MYDEF get(pub_map_u32_getter_mydef): map u32 => String = "pubmap".into();
 
-			COMPLEX_TYPE1: ::std::vec::Vec<<T as Trait>::Origin>;
```

Comment:
Is there any reason why you removed the underscores?

Reply:
Warning suppression (see other naming): not super clean but consistent.